### PR TITLE
cli: check sink uri as soon as possible.

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update key monitor metrics in both TiCDC document and official document

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

**Comment by lance6716** [comment]
please complete PR descrption. I can't get why we move a check out of `validate`

File: pkg/cmd/cli/cli_changefeed_create.go [hunk 1]
```diff
@@ -145,6 +145,10 @@ func (o *createChangefeedOptions) addFlags(cmd *cobra.Command) {
 
 // complete adapts from the command line args to the data and client required.
 func (o *createChangefeedOptions) complete(ctx context.Context, f factory.Factory, cmd *cobra.Command) error {
+	if o.commonChangefeedOptions.sinkURI == "" {
```
Comment (ben1009) [inline pkg/cmd/cli/cli_changefeed_create.go:148]: why only move this one, seems have the same logic
```
func (o *createChangefeedOptions) validateTargetTs() error {
	if o.commonChangefeedOptions.targetTs > 0 && o.commonChangefeedOptions.targetTs <= o.startTs {
		return errors.Errorf("target-ts %d must be larger than start-ts: %d", o.commonChangefeedOptions.targetTs, o.startTs)
	}
	return nil
}
```

Comment (lonng) [reply pkg/cmd/cli/cli_changefeed_create.go:148]: Please add some unit tests for it.

Comment (ben1009) [reply pkg/cmd/cli/cli_changefeed_create.go:148]: ping @3AceShowHand no update for 7 days

**Pull request closed without merging**